openslides_backend/action/actions/assignment/create_update_delete.py

```diff
@@ -16,12 +19,20 @@
 
 
 class AssignmentCreate(
+    SequentialNumbersMixin,
     CreateActionWithDependencies,
     CreateActionWithAgendaItemMixin,
     CreateActionWithListOfSpeakersMixin,
 ):
     dependencies = [AgendaItemCreate, ListOfSpeakersCreate]
 
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        instance["sequential_number"] = self.get_sequential_number(
+            instance["meeting_id"]
+        )
+        return instance
+
```

Comment:
If you overwrite the `update_instance` method in the mixin, you can save these lines in each action.

Reply:
Done.